Makefile

```diff
@@ -525,3 +527,11 @@ test-e2e-cleanup: login-required
 	$(OC_CLI) delete restore -n $(OADP_TEST_NAMESPACE) --all --wait=false
 	for restore_name in $(shell $(OC_CLI) get restore -n $(OADP_TEST_NAMESPACE) -o name);do $(OC_CLI) patch "$$restore_name" -n $(OADP_TEST_NAMESPACE) -p '{"metadata":{"finalizers":null}}' --type=merge;done
 	rm -rf $(SETTINGS_TMP)
+
+.PHONY: install-virt
```

Comment:
I would undo this change. We used a similar approach with volsync in the past

https://github.com/openshift/oadp-operator/commit/b73b5c1207a1e199403b192c23ed49f76c1f450d#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L467-L470

https://github.com/openshift/oadp-operator/commit/a76c5dc2e2ef3706e9827933f45faddf0aae2c17#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L473-L493

but it did not install through test run, only through makefile. This way, we would have two options (test run and makefile). Would stick only with test run, removing this and `deploy/virt-operator.go` script. Since uninstall logic is already written, would add it to afterAll of virt test or add it in a follow up PR, after this one merges (PREFERRED)

Reply:
Sure, I put an AfterAll to do this now instead, with a quick check to only remove it if it was the test that installed it in the first place.